### PR TITLE
Add AutoCompute to EF Core Extensions

### PR DESCRIPTION
**Pull request by lucaslorentz**

### CODE REVIEW

File: entity-framework/core/extensions/index.md [hunk 1]
```diff
@@ -72,6 +72,12 @@ With Entity Framework Core query plan debugger visualizer, you can view the quer
 
 ## Extensions
 
+### AutoCompute
```
Comment (ErikEJ) [inline entity-framework/core/extensions/index.md:75]: Suggest adding it to the bottom of the list, not the top

Comment (lucaslorentz) [reply entity-framework/core/extensions/index.md:75]: Sounds good, I will adjust it. I thought it was alphabetically sorted.

Comment (ErikEJ) [reply entity-framework/core/extensions/index.md:75]: Fair assumption, but it is completely randon at the moment.

**Comment by ErikEJ** [comment]
LGTM

File: entity-framework/core/extensions/index.md (hunk 1 above)
Comment (lucaslorentz) [reply entity-framework/core/extensions/index.md]: Updated

**Comment by ajcvickers** [comment]
Thanks!

**Pull request merged**